source/fundamentals/encrypt-fields.txt

```diff
@@ -0,0 +1,73 @@
+==============
+Encrypt Fields
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can encrypt fields in a document using a feature called
+**client-side encryption**.
+
+Client-side encryption enables your client applications to encrypt data
+*before* sending it to MongoDB, and to query documents for values in
```

Comment:
only equality lets you query for values, so perhaps just "query documents with encrypted fields"?